_includes/post_footer.html

```diff
@@ -0,0 +1,3 @@
+<div class="post-footer">
+  <a href="/articles/">View all articles</a>
```

Comment:
Actually I'd say go without because in other places, such as in the markdown of actual blog posts, we dont add a site.baseurl